os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -712,6 +722,11 @@ static int _tizenrt_create_task(struct task_struct *ptask, const char *name, u32
 	char *task_info[3];
 	priority = SCHED_PRIORITY_DEFAULT + priority;
 	priority = (priority > SCHED_PRIORITY_MAX || priority < SCHED_PRIORITY_MIN)?SCHED_PRIORITY_DEFAULT:priority;
+	if(strcmp(name, "rtw_interrupt_thread") == 0) priority = 106;
```

Comment:
Please use `strncmp` instead of `strcmp`, because of security.

Reply:
Done